### PR TITLE
PLAT-1145 Added setRequired to IDomTextualInput

### PR DESCRIPTION
**Pull request by daniel-klose**
image

### CODE REVIEW

File: platform-dom/src/main/java/com/softicar/platform/dom/input/IDomTextualInput.java [hunk 1]
```diff
@@ -91,6 +91,20 @@ public interface IDomTextualInput extends IDomFocusable, IDomValueInput<String>
...
 	IDomTextualInput setPlaceholder(IDisplayString placeholder);
 
+	// -------------------------------- required -------------------------------- //
+
+	/**
+	 * Manages the HTML required attribute.
+	 *
+	 * @param required
+	 *            whether the required attribute should be added or not (never
+	 *            <i>null</i>)
```
Comment (oliver-richers) [inline platform-dom/src/main/java/com/softicar/platform/dom/input/IDomTextualInput.java:101]: primitive `boolean` can never be null

File: platform-dom/src/main/java/com/softicar/platform/dom/input/IDomTextualInput.java [hunk 2]
```diff
@@ -91,6 +91,20 @@ public interface IDomTextualInput extends IDomFocusable, IDomValueInput<String>
...
+	 * @return this
+	 */
+	IDomTextualInput setRequired(boolean required);
+
+	boolean isRequired();
```
Comment (oliver-richers) [inline platform-dom/src/main/java/com/softicar/platform/dom/input/IDomTextualInput.java:106]: should also have a Javadoc, see `isReadonly`

**Review by oliver-richers — CHANGES_REQUESTED** [review]

File: platform-dom/src/main/java/com/softicar/platform/dom/input/IDomTextualInput.java (hunk 1 above)
Comment (daniel-klose) [reply platform-dom/src/main/java/com/softicar/platform/dom/input/IDomTextualInput.java:101]: well, I just adapted the read-only javadoc

File: platform-dom/src/main/java/com/softicar/platform/dom/input/IDomTextualInput.java (hunk 2 above)
Comment (daniel-klose) [reply platform-dom/src/main/java/com/softicar/platform/dom/input/IDomTextualInput.java:106]: see above

**Review by oliver-richers — APPROVED** [review]

**Pull request merged**